src/lib/forward.ts

```diff
@@ -133,6 +131,21 @@ export class EventForwarder {
       }
     }
 
+    {
+      const events: ZWaveNodeEvents[] = ["value added", "value notification"];
```

Comment:
If we already add the info to `value added`, we don't need to include it in `value notification` right ?

Reply:
Value notification is a special stateless one. It will never be added ;-)

Reply:
TIL